Controller/Adminhtml/Analytics/Update.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Algolia\AlgoliaSearch\Controller\Adminhtml\Analytics;
+
+class Update extends AbstractAction
+{
+    public function execute()
+    {
+        $response = $this->_objectManager->create(\Magento\Framework\DataObject::class);
+        $response->setError(false);
+
+        $this->_getSession()->setAlgoliaAnalyticsFormData($this->getRequest()->getParams());
+
+        $layout = $this->layoutFactory->create();
+        $block = $layout->createBlock('\Algolia\AlgoliaSearch\Block\Adminhtml\Analytics\Index')
```

Comment:
Can you use `::class` constant here instead of `'\Algolia\AlgoliaSearch\Block\Adminhtml\Analytics\Index'`? :)